tests/run-pass/change_current_dir.rs

```diff
@@ -0,0 +1,14 @@
+// ignore-windows: TODO the windows hook is not done yet
+// compile-flags: -Zmiri-disable-isolation
+use std::env;
+use std::path::Path;
+
+fn main() {
+    // test that `getcwd` is available
+    let cwd = env::current_dir().unwrap();
+    let parent = cwd.parent().unwrap_or(&cwd);
```

Comment:
Why not `unwrap()`? I think I am fine asserting that we are not running in the file system root... and it seems like the test would fail then anyway? Or not?

Reply:
Well I did the `unwrap_or` just for completions sake. I agree with you that probably nobody will run this in root. But changing directory to `..` is a noop if you're in the root directory (it returns 0 and everything)

Reply:
Oh that makes sense then -- seems like the test should work fine. But please add a comment explaining that.

Reply:
https://github.com/rust-lang/miri/pull/983/commits/8368d4f2b44ae1441390a2f7eee48af340c8d70c